apps/condo/domains/billing/tasks/index.js

```diff
@@ -0,0 +1,5 @@
+const sendBillingReceiptsAvailableNotificationsTask = require('./sendBillingReceiptsAddedNotificationsTask')
```

Comment:
try to translate it to English and then try to understand what this function really do

Reply:
It's better to use something like `send Reminder About New Billing Receipts`